hrui/scss/hrui.scss

```diff
@@ -40,3 +40,24 @@
 .crm-relationship-form-block-is_permission_b_a {
   display: none;
 }
+
+/*This hides some fields on the Import Page*/
+#choose-data-source{
```

Comment:
When you commit, does it run the CSS linter? I tried to remove a space before a curly bracket and it didn't allow me to commit.

![image](https://user-images.githubusercontent.com/3973243/45479167-a2fa4980-b73d-11e8-88df-d4d4e460138f.png)

Let's ensure your linter works before you create a PR Feedback commit.